distributor/http.go

```diff
@@ -0,0 +1,56 @@
+package distributor
```

Comment:
Don't mind either way.  The ingester_client.go and ingester/http.go are going to be deleted in the next PR.